### PR TITLE
SlimeRead: Fix "Unexpected JSON token" on pageList and choose apiUrl randomly

### LINKED ISSUE #1830: SlimeRead: Unexpected Json token
### Source information

SlimeRead 1.4.4

### Source language

Português 

### Steps to reproduce

1 install The extension
2 Open The extension
3 open Any manga

### Expected behavior

Screenshot_2024-03-11-21-44-23-356_app mihon

### Actual behavior

Screenshot_2024-03-11-21-44-23-356_app mihon

### Mihon/Tachiyomi version

0.16.4

### Android version

Android 13

### Other details

_No response_

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] I have updated the app to version **[0.15.3](https://github.com/mihonapp/mihon/releases/latest)**.
- [X] I have updated all installed extensions.
- [X] I have tried the [troubleshooting guide](https://tachiyomi.org/help/guides/troubleshooting/).
- [ ] I have admitted that I am a clown by having checked this box, as I have not read these acknowledgements.
- [X] If this is an issue with the app itself, I should be opening an issue in the [app repository](https://github.com/tachiyomiorg/tachiyomi/issues/new/choose).
- [X] I will fill out all of the requested information in this form.

### PR DESCRIPTION
**Pull request by bapeey**
And update apiUrl (~again~ actually, they choose the api randomly) 
Closes #1830 

Checklist:

- [x] Updated `extVersionCode` value in `build.gradle` for individual extensions
- [ ] Updated `overrideVersionCode` or `baseVersionCode` as needed for all multisrc extensions
- [x] Referenced all related issues in the PR body (e.g. "Closes #xyz")
- [ ] Added the `isNsfw = true` flag in `build.gradle` when appropriate
- [x] Have not changed source names
- [ ] Have explicitly kept the `id` if a source's name or language were changed
- [x] Have tested the modifications by compiling and running the extension through Android Studio
- [ ] Have removed `web_hi_res_512.png` when adding a new extension

### CODE REVIEW

**Comment by AwkwardPeak7** [comment]
rebase on main

File: src/pt/slimeread/src/eu/kanade/tachiyomi/extension/pt/slimeread/SlimeRead.kt [hunk 1]
```diff
@@ -30,7 +31,9 @@ class SlimeRead : HttpSource() {
 
     override val baseUrl = "https://slimeread.com"
 
-    private val apiUrl = "https://old.slimeread.com:8443"
+    private val apiList = arrayOf("free", "beta", "api", "data", "test", "dev", "staging", "prod", "old", "new")
```
Comment (stevenyomi) [inline src/pt/slimeread/src/eu/kanade/tachiyomi/extension/pt/slimeread/SlimeRead.kt]: Inline this. You can use `run {}` to create a block.

File: src/pt/slimeread/src/eu/kanade/tachiyomi/extension/pt/slimeread/SlimeRead.kt [hunk 2]
```diff
@@ -30,7 +31,9 @@ class SlimeRead : HttpSource() {
...
+
+    private val apiUrl = "https://${apiList.random()}.slimeread.com:8443"
```
Comment (stevenyomi) [inline src/pt/slimeread/src/eu/kanade/tachiyomi/extension/pt/slimeread/SlimeRead.kt]:
```suggestion
    private val apiUrl = run {
        val apiList = arrayOf("free", "beta", "api", "data", "test", "dev", "staging", "prod", "old", "new")
        "https://${apiList.random()}.slimeread.com:8443"
    }
```

**Comment by nomaxsnx** [comment]
Screenshot_2024-03-14-15-48-45-537_app mihon

**Pull request merged**